.github/workflows/readme-check.yml

```
@@ -0,0 +1,26 @@
+name: Check README Updates
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  workflow_dispatch:
```

```suggestion
  workflow_dispatch:
```
@iulianbarbu do we really need this option here?

---

FYI @virajbhartiya , my suggestion for this line specifically can be ignored, so you can remove it.